src/libtilt/projection/project_fourier.py

```diff
@@ -107,3 +92,49 @@ def extract_central_slices_rfft(
     # take complex conjugate of values from redundant half transform
     projections[conjugate_mask] = torch.conj(projections[conjugate_mask])
     return projections
+
+def compute_vol_dtf( #TODO: Is this the best place to have this?
```

Comment:
```suggestion
def _compute_dtf(
```

Can we go with the above as a temporary API for now until we figure out a name and where this should go etc?

Reply:
Ok. Let's keep it as it is. However, I would rather keep the VOL part in the name since it works only for 3D data.